definitions/pipelines/Chipseq_alignment.cwl

```diff
@@ -0,0 +1,148 @@
+        type: ../types/labelled_file.yml#labelled_file[]?
+        default: []
+    per_target_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]?
+        default: []
+    summary_intervals:
+        type: ../types/labelled_file.yml#labelled_file[]?
+        default: []
+    picard_metric_accumulation_level:
+        type: string
+    bqsr_intervals:
+        type: string[]
+    minimum_mapping_quality:
+        type: int?
+    minimum_base_quality:
+        type: int?
+outputs:
+    bam:
+        type: File
+        outputSource: alignment/final_bam
+    mark_duplicates_metrics:
+        type: File
+        outputSource: alignment/mark_duplicates_metrics_file
+    insert_size_metrics:
+        type: File
+        outputSource: qc/insert_size_metrics
+    insert_size_histogram:
+        type: File
+        outputSource: qc/insert_size_histogram
+    alignment_summary_metrics:
+        type: File
+        outputSource: qc/alignment_summary_metrics
+    gc_bias_metrics:
+        type: File
+        outputSource: qc/gc_bias_metrics
+    gc_bias_metrics_chart:
+        type: File
+        outputSource: qc/gc_bias_metrics_chart
+    gc_bias_metrics_summary:
+        type: File
+        outputSource: qc/gc_bias_metrics_summary
+    wgs_metrics:
+        type: File
+        outputSource: qc/wgs_metrics
+    flagstats:
+        type: File
+        outputSource: qc/flagstats
+    verify_bam_id_metrics:
+        type: File
+        outputSource: qc/verify_bam_id_metrics
+    verify_bam_id_depth:
+        type: File
+        outputSource: qc/verify_bam_id_depth
+    per_base_coverage_metrics:
+        type: File[]
+        outputSource: qc/per_base_coverage_metrics
+    per_base_hs_metrics:
+        type: File[]
+        outputSource: qc/per_base_hs_metrics
+    per_target_coverage_metrics:
+        type: File[]
+        outputSource: qc/per_target_coverage_metrics
+    per_target_hs_metrics:
+        type: File[]
+        outputSource: qc/per_target_hs_metrics
+    summary_hs_metrics:
+        type: File[]
+        outputSource: qc/summary_hs_metrics
+    bamcoverage_bigwig:
+        type: File
+        outputSource: qc/bamcoverage_bigwig
+    tag_directory:
+        type: Directory
+        outputSource: homer_tag_directory/tag_directory
+
+steps:
+    alignment:
+        run: ../subworkflows/sequence_to_bqsr.cwl
+        in:
+            reference: reference
+            unaligned: chipseq_sequence
+            mills: mills
+            known_indels: known_indels
+            dbsnp_vcf: dbsnp_vcf
+            bqsr_intervals: bqsr_intervals
+            final_name: final_name
+        out: [final_bam,mark_duplicates_metrics_file]
+
+    bam_to_sam:
+        run: ../tools/bam_to_sam.cwl
+        in:
+            bam: alignment/final_bam
+        out: [final_sam]
+
+    homer_tag_directory:
+        run: ../tools/homer_tag_directory.cwl
+        in:
+            sam: bam_to_sam/final_sam
+            reference: reference
```

Comment:
Since the `reference` was removed from `homer_tag_directory.cwl`, it needs to be removed here, too.

Reply:
Sorry about that! Just fixed it.